CHANGELOG.rst

```diff
@@ -14,6 +14,10 @@ Change Log
 Unreleased
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+[1.6.0] - 2024-01-31
```

Comment:
Note that this typically requires the version update in code as well. See https://github.com/openedx/code-annotations/blob/764c306d859036971177457cb08cc04e7ee2989c/code_annotations/__init__.py#L5. Thanks.